src/spikeinterface/core/zarrextractors.py

```diff
@@ -72,7 +72,7 @@ def __init__(self, folder_path: Path | str, storage_options: dict | None = None)
             time_kwargs = {}
             time_vector = self._root.get(f"times_seg{segment_index}", None)
             if time_vector is not None:
-                time_kwargs["time_vector"] = time_vector
+                time_kwargs["time_vector"] = time_vector[:]
```

Comment:
Maybe I'm misunderstanding the compression-decompression, but how does switching to a view change this?

Reply:
Without the `[:]` the time_vector is set to a `zarr.Array` object.

This object performs a decompression/retrieval from file upon slicing, so if I call `get_times()` 100 times, the data will be accessed from the file (and optionally decompressed) 100 times.

This simple change slices the array, so that `time_kwargs["time_vector"]` is now a numpy array and not a zarr object anymore.

Hope this is clear :)

Reply:
Oh yeah that makes sense. Cool.